meadServer/app.js

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
You should clean up the local DB comments as a forcing function to set up your config files correctly. There should be a way to set up your config to do this without special case logic